src/main/resources/webapp/js/controllers.js

```diff
@@ -1265,6 +1265,12 @@
             }, 10);
         };
 
+        $scope.getUploadSuccessMessage = function () {
+            if ($scope.form.dto && $scope.form.dto.actualDate) {
+                $scope.rescheduleModalBody = $scope.msg('prevac.visitReschedule.actualDateUpdateSuccessful');
```

Comment:
I think you should add else here, because if you use it first to add actual date to visit and later to change planned date it can show actual date updated message

Reply:
I've tested some cases and everything was correct. But I think I can add else here just to be sure. Thanks.